main/src/cgeo/geocaching/activity/OAuthAuthorizationActivity.java

```diff
@@ -64,6 +64,8 @@
     private ProgressDialog requestTokenDialog = null;
     private ProgressDialog changeTokensDialog = null;
 
+    private String lastVerifier = "";
```

Comment:
```suggestion
    private String lastVerifier = null;
```
Otherwise we behave different if the first verifier is `""`.

Reply:
I think in general I got different behavior here. Original code was returning an error in case of empty verifier. SO probably we want to keep it(?). Thus, I changed the code to maintain the same behavior, so it doesn't matter actually what is the initial value here. But changed it to null, don't see any issue here.